docs/testnet-guardian-runbook.md

```diff
@@ -0,0 +1,152 @@
+# Testnet-Guardian Runbook
+
+## Overview
+
+A testnet guardian is the point of contact for anything engineering-related for an upcoming or a running testnet. Broadly speaking, a testnet guardian would perform/coordinate tasks involving building, testing and deploying testnet(s) for a release. They would also serve as a point-of-contact for protocol/product queries related to the testnet challenges from the community. This document describes all the tasks that a testnet guardian would perform during a release cycle and describes possible actions  in the case of events that affect the testnet. The role of a testnet guardian spans over a release cycle starting from code-freeze from the engineering team to the end of the release. The initial phase between code freeze and the public release is the QA phase where the testnet guardian will be responsible for building, deploying and testing internal QA-nets. On successful completion of the QA phase, the release phase starts during which the testnet guardian would support the release process and be the point of contact between the community members.
+
+## QA-net
+
+The role of a testnet guardian for a release begins after the code-freeze. The following is the list of tasks that a testnet guardian would be responsible for:
+
+### Release branch
+
+Confirm with the protocol and the product team that all the required PRs are in. Cut a release branch after the code-freeze(The naming convention for beta-testnets is `release/0.0.<release_number>-beta`. For example, `release/0.0.13-beta`)
+
+### Build
+
+The latest commit on the release branch should have successful builds for both macOS and Linux.
+Go to the CI workflow for the specific commit and look for `build- macos` and `build-artifacts--testnet_postake_medium_curves` jobs for mac and linux builds respectively.
+
+##### Publish the location of the binaries
+Publish the locations to the binaries on the discord channel `#QA-net details` channel for the rest of the team
```

Comment:
Not feedback, but curious what this step is for.  Is there a follow up action by the team or someone else that needs to be done with this information>?

Reply:
This is for the team to run a node on their machines and connect to the testnet. Usually, one has to look for the path from one of the CI tasks and then download.